src/UI/templates/default/Popover/popover.less

```diff
@@ -0,0 +1,5 @@
+.il-popover {
+  .webui-popover-content>div:first-child {
+    width: 100%;
+  }
+}
```

Comment:
Introduce src/UI/templates/libs/node_modules/webui-popover/src/jquery.webui-popover.less here to the build process. 

Maybe even use a custom version of this less and promote some of the variables to the global variables.less